src/pages/documentation/adobe-media-analytics/index.md

```diff
@@ -2,6 +2,11 @@ import Tabs from './tabs/index.md'
 
 # Adobe Analytics - Media Analytics for Audio and Video
 
+<InlineAlert variant="success" slots="header, text"/>
+Media for Edge Network extension is now available! 
+
+Refer to the [Migrating to Media for Edge Network Guide](migration-guide.md) and [Media for Edge Network extension](../media-for-edge-network/index.md) documentation to learn more about implementing Media Analytics using Adobe Experience Edge.
```

Comment:
Reword to remove "Analytics", "... implementing Streaming Media using Adobe Experience Edge Network."